packages/utils/src/formattingSpace.ts

```diff
@@ -62,11 +69,25 @@ export function removeFormattingSpace(node: Node): string {
     //    spaces (U+3000) whose white-space value collapses spaces is
     //    removed.
     if (_isAtSegmentBreak(node, 'end')) {
-        const endSpace = /[ \u3000]*$/g;
-        newText = newText.replace(endSpace, '');
+        newText = newText.replace(_endSpace, '');
     }
     return newText;
 }
+/**
+ * Return true if the given node is immediately folowing a space inside the same inline context,
+ * to see if its frontal space must be removed.
+ *
+ * @param {Element} node
+ * @returns {boolean}
+ */
+function _followInlineSpace(node: Node): boolean {
```

Comment:
Since the other is `_isAtSegmentBreak` then I guess this one should be`_followsInlineSpace` instead ?

Reply:
:+1: